### PR TITLE
feat(crux): cal.com template

### PR DESCRIPTION
**Pull request by robot9706**
- Added the `cal.com` template, which uses the custom build image from `ghcr.io/dyrector-io`

Image is built from `https://github.com/calcom/docker` using the `calcom` service as an example from the `docker-compose.yaml` file.

Depends on #456 which fixes the default deployment prefix.

### CODE REVIEW

File: web/crux/templates/cal-com.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            "value": "1"
+          },
+          {
+            "key": "DATABASE_URL",
+            "value": "postgresql://cal-user:{POSTGRES_PASSWORD}@cal-db/cal-db"
+          }
+        ],
+        "secrets": [
+          {
+            "key": "NEXTAUTH_SECRET",
```
Comment (polaroi8d) [inline web/crux/templates/cal-com.json:104]: Need some guidance for these secrets in the description.

File: web/crux/templates/cal-com.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  "registries": [
+    {
+      "name": "Docker Hub Library",
+      "type": "hub",
+      "hub": {
+        "imageNamePrefix": "library"
+      }
+    },
+    {
+      "name": "dyrector.io GitHub registry",
```
Comment (polaroi8d) [inline web/crux/templates/cal-com.json:13]: Use our Docker HUB registry instead of GitHub.

**Review by polaroi8d — CHANGES_REQUESTED** [review]

File: web/crux/templates/cal-com.json [hunk 3]
```diff
@@ -0,0 +1,115 @@
+{
+  "name": "cal.com",
+  "description": "The open-source Calendly alternative. Cal.com is the event-juggling scheduler for everyone. Focus on meeting, not making meetings. Free for individuals. Before deployment make sure to set the 'POSTGRES_PASSWORD' secret and update the 'DATABASE_URL' with the password, as well as set the 'NEXTAUTH_SECRET' and 'CALENDSO_ENCRYPTION_KEY' secrets to a secure random string (16-32 characters should suffice, for example use 'openssl rand -base64 32'). If you'd like to run the app on a port other than 7000, update the exposed port of the 'cal-com' container as well as the 'PORT' and 'NEXT_PUBLIC_WEBAPP_URL' environment variables.",
```
Comment (nandor-magyar) [inline web/crux/templates/cal-com.json:3]: Filling ingress name + host and removing external ports are the wae for exposing it to a public domain, that requires traefik. These would be nice to have.

Comment (robot9706) [reply web/crux/templates/cal-com.json]: Removed ports, added ingress settings. Also modified the description so it shows how to change the domain as well as it tells the user that the template requires Traefik from the agent :)

**Review by nandor-magyar — APPROVED** [review]

**Pull request merged**